schema/shared/src/main/scala/zio/blocks/schema/Reflect.scala

```diff
@@ -275,6 +275,15 @@ sealed trait Reflect[F[_, _], A] extends Reflectable[A] { self =>
 
     loop(this, 0).asInstanceOf[Option[Reflect[F, A]]]
   }
+
+  def aspect[Min >: A, Max <: A](aspect: SchemaAspect[Min, Max]): Reflect.Bound[A] =
+    aspect(self.asInstanceOf[Reflect.Bound[A]])
+
+  def aspect[B](part: Optic[A, B], aspect: SchemaAspect[B, B]): Reflect.Bound[A] =
```

Comment:
```suggestion
  def aspect[B, Min >: B, Max <: B](part: Optic[A, B], aspect: SchemaAspect[Min, Max]): Reflect.Bound[A] =
```

This is more precise and will allow "recursive" schema aspects to be applied starting from a certain location.

Reply:
See also comment about `IsBinding` type class.